content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -1,82 +1,87 @@
 ---
-title: 'Deploying to AWS Lambda'
-metaTitle: 'Deploying to AWS Lambda'
-metaDescription: 'Learn how to deploy Node.js and TypeScript applications that are using Prisma Client to AWS Lambda.'
+title: 'Deploy to AWS Lambda'
+metaTitle: 'Deploy to AWS Lambda'
+metaDescription: 'Learn how to deploy Node.js and TypeScript applications based on Prisma Client to AWS Lambda.'
 ---
 
 <TopBlock>
 
-In this guide, you will set up and deploy a serverless Node.js REST API to [AWS Lambda](https://aws.amazon.com/lambda/) using the Serverless Framework.
+This guide explains how to set up and deploy a serverless Node.js REST API to [AWS Lambda](https://aws.amazon.com/lambda/) with the Serverless Framework.
 
-AWS Lambda is part of the AWS cloud platform and allows you to use the serverless paradigm to run your code without managing servers. To deploy a REST API to AWS Lambda, you need to make use of several additional AWS services, namely [S3](https://aws.amazon.com/s3/) to host the files and [API Gateway](https://aws.amazon.com/api-gateway/) to expose the API over HTTP.
+AWS Lambda is part of the AWS cloud platform. You can use it to run your code in a serverless environment. To deploy a REST API to AWS Lambda, you must also use [S3](https://aws.amazon.com/s3/) to host the files and [API Gateway](https://aws.amazon.com/api-gateway/) to expose the API over HTTP.
 
-The Serverless Framework simplifies the deployment to Lambda with a CLI that helps with workflow automation and AWS resource provisioning.
+The [Serverless Framework](https://www.serverless.com/) simplifies deployment to Lambda. It provides a CLI that helps with workflow automation and AWS resource provisioning.
 
-The REST API will use Prisma Client to handle fetching, creating, and deleting records from a database. Specifically, each function will represent a REST resource endpoint and use Prisma Client to handle database operations against a PostgreSQL database (e.g. hosted on Heroku).
+This guide uses the [Prisma AWS example](https://github.com/prisma/prisma-examples/tree/latest/deployment-platforms/aws-lambda), which has example REST endpoints preconfigured as serverless functions. The Prisma AWS example uses Prisma Client to fetch, create, and delete records from a database. Each function represents a REST resource endpoint and uses Prisma Client to handle database operations against a PostgreSQL database hosted on a platform such as Supabase.
```

Comment:
I don't think we should have a comma here, because it implies that the sentence is complete without the clause after the comma. So it would imply ".... a PostgreSQL database hosted on a platform." That doesn't make sense. So I think we should omit the comma here.

However, now that I look at this again, I think we should omit "Supabase" here and say "an xyz platform" where xyz is the type of platform we're talking about. It isn't very helpful to just name one specific platform in this intro, even if the guide only covers that platform, because the principle can be extended to other platforms.